v2/java/README.md

```diff
@@ -3,25 +3,25 @@
 ## Overview
 This program demonstrates the capability of the Yelp API version 2.0
 by using the Search API to query for businesses by a search term and location,
-and the Business API to query additional information about the top result
-from the search query.
+and the Business API to query additional information about the highest ranked
```

Comment:
Highest ranked is correct, good.
("highest rated" is a little more complicated.)